requirements.txt

```diff
@@ -0,0 +1,6 @@
+pyYaml==5.3.1
```

Comment:
again, I don't want those version pinned in CCM
also pytest isn't a dependency for ccm, it's `test_require` part not the `install_requires` part

Reply:
I removed the version from each package.
I fixed the dependency